lib/matplotlib/figure.py

```diff
@@ -1817,12 +1817,12 @@ def text(self, x, y, s, fontdict=None, withdash=False, **kwargs):
         s : str
             The text string.
 
-        fontdict : dictionary, optional, default: None
-            A dictionary to override the default text properties. If fontdict
-            is None, the defaults are determined by your rc parameters. A
-            property in *kwargs* override the same property in fontdict.
+        fontdict : dict, optional
+            A dictionary to override the default text properties. If not given,
+            The defaults are determined by your rc parameters. A property
+            in *kwargs* override the same property in fontdict.
```

Comment:
overrideS
Or plural everywhere ("Properties given... override the corresponding properties...")
italicize fontdict.